### ISSUE TITLE
Error fetching submodules

### ISSUE THREAD

**Issue by schustan** [lcpz/awesome-copycats#289]
Cloning does not work as described in the README.

```
> git clone --depth 1 --recursive https://github.com/lcpz/awesome-copycats.git
Cloning into 'awesome-copycats'...
remote: Enumerating objects: 409, done.
remote: Counting objects: 100% (409/409), done.
remote: Compressing objects: 100% (369/369), done.
remote: Total 409 (delta 54), reused 370 (delta 38), pack-reused 0
Receiving objects: 100% (409/409), 18.62 MiB | 8.67 MiB/s, done.
Resolving deltas: 100% (54/54), done.
Submodule 'freedesktop' (https://github.com/lcpz/awesome-freedesktop.git) registered for path 'freedesktop'
Submodule 'lain' (https://github.com/lcpz/lain.git) registered for path 'lain'
Cloning into '/home/nimra/awesome-copycats/freedesktop'...
remote: Enumerating objects: 407, done.        
remote: Counting objects: 100% (10/10), done.        
remote: Compressing objects: 100% (8/8), done.        
remote: Total 407 (delta 3), reused 6 (delta 2), pack-reused 397        
Receiving objects: 100% (407/407), 188.94 KiB | 833.00 KiB/s, done.
Resolving deltas: 100% (208/208), done.
Cloning into '/home/nimra/awesome-copycats/lain'...
remote: Enumerating objects: 6235, done.        
remote: Counting objects: 100% (75/75), done.        
remote: Compressing objects: 100% (57/57), done.        
remote: Total 6235 (delta 38), reused 38 (delta 18), pack-reused 6160        
Receiving objects: 100% (6235/6235), 1.54 MiB | 5.68 MiB/s, done.
Resolving deltas: 100% (3797/3797), done.
fatal: remote error: upload-pack: not our ref e4b75661c30a0525f318a3e15d623c201242e0a7
Fetched in submodule path 'freedesktop', but it did not contain e4b75661c30a0525f318a3e15d623c201242e0a7. Direct fetching of that commit failed.
```

This can be worked-around by `git submodule update --force --recursive --init --remote`, but should be referenced correctly in the repo.

Best regards

**Comment by Pocco81** [comment]
> This can be worked-around by git submodule update --force --recursive --init --remote, but should be referenced correctly in the repo.

Can confirm it works.

**Comment by Kanahmal** [comment]
This worked for me as well, I'll mention to any new folks that you have to cd into the awesome copycats directory before running the git submodules (I never used git submodules before).

**Comment by lcpz** [comment]
@schustan Thank you for reporting the issue. I fixed the command, it requires `git >= 2.23` (Q3 2019). I hope it's OK.
